test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/CpsPackageReferenceProjectTests.cs

```diff
@@ -4125,6 +4125,31 @@ public async Task GetInstalledAndTransitivePackagesAsync_MutipleProjectReference
             }
         }
 
+        [Fact]
+        public async Task GetInstalledAndTransitivePackagesAsync_ProjectReferenceWithNullFramework_EmptyInstalledAndTransitivePackagesAsync()
+        {
+            // Project2 -> Project1 ->  PackageA (1.0.0) -> PackageB (1.0.0)
+
+            // Arrange
+            using var rootDir = new SimpleTestPathContext();
+
+            await CreatePackagesAsync(rootDir, packageAVersion: "1.0.0");
+
+            PackageSpec prj1Spec = ProjectTestHelpers.GetPackageSpec("Project1", rootDir.SolutionRoot, framework: "netstandard2.0", dependencyName: "PackageA");
+            PackageSpec prj2Spec = ProjectTestHelpers.GetPackageSpec("Project2", rootDir.SolutionRoot, framework: "netstandard2.0").WithTestProjectReference(prj1Spec, null);
+
+            await RestorePackageSpecsAsync(rootDir, output: null, prj1Spec, prj2Spec);
+
+            CpsPackageReferenceProject project2 = PrepareCpsRestoredProject(prj2Spec);
+
+            // Act
+            ProjectPackages result = await project2.GetInstalledAndTransitivePackagesAsync(includeTransitiveOrigins: true, CancellationToken.None);
+
+            // Assert
+            Assert.Empty(result.InstalledPackages); // No installed packages
```

Comment:
So there's no project 2 packages despite the project showing them as such?

Reply:
I'd love to understand the details beyond this test case.

Reply:
Deleting this change, I was wrong about the null framework, it's in the assets file when it's null